### DISCUSSION TITLE
Markdown Editor built with alpine & tailwind

### DISCUSSION THREAD

**Discussion by JeremyJaydan** [alpinejs/alpine#2035] [2. Show and tell]
A few years ago I made a little markdown editor that I use pretty much everyday as a temporary editor for taking notes.

I spent the last two days re-writing it with Alpine + tailwind: https://md.jeremyjaydan.dev

Before it did have auto-save with localstorage  but it wasn't reliable and I also wanted the ability to save multiple notes. Alpine made this super easy with `$persist`.

Most of the time re-writing this project was spent on the preview caret (for the preview auto-scroll), it's a little buggy but it works for the most part. :)

**Comment by bora89** [comment]
Hi Jeremy, it looks clean and attractive, nice!

**Reply by JeremyJaydan** [reply]
Thank you! :)

**Comment by ibra379** [comment]
> _Gyh**fghhhh**_
